tests/014-test-create-config-docker.sh

```diff
@@ -0,0 +1,47 @@
+source $SRCDIR/libtest.sh
+
+# Test creation of default (docker) config.
+# Returns 0 on success and 1 on failure.
+test_docker_config() {
+  local devs=$TEST_DEVS
+  local test_status=1
+  local testname=`basename "$0"`
+  local vg_name="css-test-foo"
+  local infile="/etc/sysconfig/docker-storage-setup"
+  local outfile="/etc/sysconfig/docker-storage"
+  local default_config_name="docker"
```

Comment:
Ok I was reading your other code and saw 
cleanup ... $default_config_name 

And my brain translated this to 
rm -f $default_config_name